specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9155 @@
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeStatus"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_Status": {
+            "$ref": "./examples/IntegrationRuntimes_Status.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}/:listAuthKeys": {
+      "post": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Gets integrationRuntimeAuthKeys",
+        "operationId": "IntegrationRuntimes_ListAuthKeys",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeAuthKeys"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_ListAuthKeys": {
+            "$ref": "./examples/IntegrationRuntimes_ListAuthKeys.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}/:regenerateAuthKey": {
+      "post": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Regenerates integrationRuntimeAuthKey",
+        "operationId": "IntegrationRuntimes_RegenerateAuthKey",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeRegenerateKeyParameters"
+            }
+          }
```

Comment:
This body parameter is not marked "required: true". That is a common oversight. How does this operation behave when no body is passed?

Reply:
changed

Reply:
The operation returns a http 500 when no body is passed